content/applications/websites/website/web_design/building_blocks/inner_content_blocks.rst

```diff
@@ -0,0 +1,39 @@
+=============================
+Inner content building blocks
+=============================
+
+You can use the :guilabel:`Inner content` building blocks, such as the :guilabel:`Button`,
+:guilabel:`Image`, :guilabel:`Video`, or  :ref:`Social Media <inner_content_blocks/social_media>`
+blocks, to enhance your website's view by providing a structure and design for the content available
+on your web pages. Select a block from the :guilabel:`Inner content` section in the
+:guilabel:`Blocks` tab of the website builder, and drag and drop it onto your web page.
```

Comment:
I think there's too much information in the first sentence when it should provide "this is what we're talking about" 
Here's a rewriting suggestion, I just separated into two paragraphs to make it a bit clearer:
```suggestion
The inner content building blocks allow you to add elements such as videos, images, buttons,
:ref:`Social Media <inner_content_blocks/social_media>` redirection buttons, and more into
pre-existing structure blocks.

To do so, access a website page and open the website builder by clicking :guilabel:`Edit`. Then,
drag and drop a block from the :guilabel:`Inner content` section of the :guilabel:`Blocks` tab onto
your web page.
```
I'd remove the guilabel when talking about images and stuff because this is more the concept, readers don't know yet where to look so they don't see image or video written on their screen